### PR TITLE
Support 'report' keyword

### PR DESCRIPTION
**Pull request by kinbiko**
You saw this happen. Be strict!

### CODE REVIEW

File: src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java [hunk 1]
```diff
@@ -43,22 +56,56 @@
             "        ║ 2018-01-02 │ DEBIT            │ £0.72  │ £1.78   │ Paid bill   ║\n" +
             "        ╚════════════╧══════════════════╧════════╧═════════╧═════════════╝\n";
 
+    private TestLogger logger = new TestLogger();
+    private TestTransactionMaker testTransactionMaker = new TestTransactionMaker();
+    private Application target = new Application(logger, testTransactionMaker);
+
     @Test
-    public void printsHelpMessageIfHelpIsGiven() {
+    public void givenHelpLogsHelpMessage() {
```
Comment (perrymant) [inline src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java:64]: Updated name to be more uniform with the others.

File: src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java [hunk 2]
```diff
@@ -43,22 +56,56 @@
...
         target.start(new String[]{"help"});
-        Assert.assertEquals(HELP_MESSAGE, logger.getMessage());
+        assertEquals(HELP_MESSAGE, logger.getMessage());
     }
 
     @Test
-    public void doesntPrintHelpWithNoArgs() {
+    public void givenNoArgsLogsErrorMessage() {
```
Comment (perrymant) [inline src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java:70]: Updated name to be more uniform with the others

**Review by perrymant — APPROVED** [review]

**Comment by perrymant** [comment]
LGTM

**Comment by kinbiko** [comment]
I don't mind the changes that you made here - I agree with them all - but in the future do **not** make commits to other people's work unless they've given permission first.

- What if they've got more commits on their branch locally that they haven't pushed yet? You've probably just given them a merge conflict they didn't ask for.
- A lot of engineers (most in fact) can't sufficiently separate themselves from their code, and might take offence at someone correcting their code without checking with them first.
- As a reviewer it is very rare that you will have more knowledge and context about a PR than the author. Thus, it's safer to point out potential issues than to fix them yourself.
- You're creating more work because the person with the most context originally now has to review the work of someone with less context.

Just something to be mindful about in the future.

**Pull request merged**